app/index.jsx

```diff
@@ -227,7 +227,7 @@ class App extends Component {
     };
     changeGranularity = timePeriod => this.setState({ granularity: timePeriod });
     changeChartType = chartType => this.setState({ chartType });
-
+    handleStateChange = (tag, option) => console.log(`chart state changed to ${tag} with the option of ${option ? JSON.stringify(option) : '{}'}`)
```

Comment:
Do we want to keep to keep this console.log?

Reply:
@zaki-binary yeah, actually that's for test and like settingChange, I just want to see if anything work or not